echidna/scripts/zero_nu_limit.py

```diff
@@ -23,56 +23,80 @@
 import echidna.limit.limit_config as limit_config
 import echidna.limit.limit_setting as limit_setting
 import echidna.limit.chi_squared as chi_squared
+import echidna.output.plot_chi_squared as plot_chi_squared
 
 
 if __name__ == "__main__":
+    import argparse
+    
+
+    parser = argparse.ArgumentParser(description="Example limit setting script")
+    parser.add_argument("-v", "--verbose", action="store_true",
+                        help="Print progress and timing information")
+    args = parser.parse_args()
+
     # Create signal spectrum
     Te130_0n2b = store.load(echidna.__echidna_home__ +
-                            "data/TeLoadedTe130_0n2b.ntuple_reco.hdf5")
+                            "/data/Te130_0n2b_mc_smeared.hdf5")
+    print Te130_0n2b._name
 
     # Create background spectra
     Te130_2n2b = store.load(echidna.__echidna_home__ +
-                            "data/TeLoadedTe130_2n2b.ntuple_reco.hdf5")
-    B8 = store.load(echidna.__echidna_home__ +
-                    "data/TeLoadedB8.ntuple_reco.hdf5")
+                            "/data/Te130_2n2b_mc_smeared.hdf5")
+    print Te130_2n2b._name
+    B8_Solar = store.load(echidna.__echidna_home__ +
+                          "/data/B8_Solar_mc_smeared.hdf5")
+    print B8_Solar._name
 
     # Shrink spectra to 5 years - livetime used by Andy
     # And make 3.5m fiducial volume cut
+    # Temporary fix to _num_decays and _raw_events
+    Te130_0n2b._num_decays = Te130_0n2b.sum()
+    Te130_0n2b._raw_events = 200034
```

Comment:
Unsure what this is for. num_decays stores number of particle decays and raw_events stores the number of events which were used to produce the spectra. They are different due to fitters etc and shouldn't be modified in my opinion. If it is for scaling then we need a function which will scale_by_count(spectra, count). I don't think counts should be stored in the spectra object, it should be passed by argument at the start of the script